src/app/containers/Blocks/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { objectOf, arrayOf, func, shape, string, any } from 'prop-types';
+
+// Inlined as this is a temporary component
+const BlockString = props => {
+  const stringProps = JSON.stringify(props);
```

Comment:
CodeClimate is erroring since the case where `BlockString` is used isn't captured in the unit tests.

Reply:
This test case is in the unit test, so this PR shouldn't be blocked because of that.